data/eos-metrics-event-recorder.service.in

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=EndlessOS Metrics Event Recorder Server
-Wants=systemd-logind.service
```

Comment:
I'm a little confused by when you would use "wants" now. I thought this was correct as written because the event recorder daemon, as written, does want the login manager to start but doesn't require that it does or care whether its started before or after the login manager.